static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -341,7 +347,8 @@ CStudioAuthoring.Module.requireModule(
 
           const external = {
             acecode: '/studio/static-assets/js/tinymce-plugins/ace/plugin.min.js',
-            paste_cleanup: '/studio/static-assets/js/tinymce-plugins/paste_cleanup/plugin.js'
+            paste_cleanup: '/studio/static-assets/js/tinymce-plugins/paste_cleanup/plugin.js',
+            paste_text: '/studio/static-assets/js/tinymce-plugins/paste_text/plugin.js',
```

Comment:
Please make sure you're running prettier to format code on any modified files. We don't use trailing commas.
```suggestion
            paste_text: '/studio/static-assets/js/tinymce-plugins/paste_text/plugin.js'
```